deploy/catalog_resources/community/7.8/kiecloud-operator.7.8.0.clusterserviceversion.yaml

```diff
@@ -440,7 +440,7 @@ spec:
     name: mysql-57-rhel7
   - image: registry.redhat.io/rhscl/postgresql-10-rhel7:latest
     name: postgresql-10-rhel7
-  - image: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.3
+  - image: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.5
```

Comment:
closer... but we still need the 1.3 image tag in the relatedImages section... we need to modify code to add both images here.